daprdocs/content/en/reference/components-reference/supported-bindings/cron.md

```diff
@@ -59,6 +59,20 @@ For ease of use, the Dapr cron binding also supports few shortcuts:
 * `@every 15s` where `s` is seconds, `m` minutes, and `h` hours
 * `@daily` or `@hourly` which runs at that period from the time the binding is initialized
 
+## Listen to the cron binding
+
+After setting up the cron binding, all we need to do is listen on an endpoint that matches the name of our component. let's assume the <NAME> is `scheduled`. Note that this will be made as a HTTP `POST` request. The example below shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.
```

Comment:
Hi @swuecho,

Thank you for this contribution. Some minor suggestions based on docs style and tone guidance: https://docs.dapr.io/contributing/contributing-docs/#style-and-tone

 ```suggestion
After setting up the cron binding, all you need to do is listen on an endpoint that matches the name of your component. Assume the <NAME> is `scheduled`. Note that this will be made as a HTTP `POST` request. The below example shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.
```